tests/test_helper/ledger.bash

```diff
@@ -1,44 +1,40 @@
 PEM_ROOT="$(mktemp -d)"
 
-function ledger() {
-    local pem_arg
-    local error
-    local check_balance
+# Do not rename this function `ledger`.
+# It clashes with the call to the `ledger` binary on CI
```

Comment:
Good to know. Thanks.

However, I would leave it this way since it's future-proof. Those naming clashes between the bash functions and native environment are NOT fun to debug.